bluesky-social/social-app
# Account quick switch on web

Clicking the profile picture in the top left now shows an account quick switch menu

Animation is disabled for prefers-reduced-motion


https://github.com/user-attachments/assets/ea67c131-3038-459f-b36b-e3442bcd6010



# Test plan

- Click all the buttons
- Check breakpoints
